frontend/views/pages/Join.vue

```diff
@@ -145,8 +145,8 @@ export default {
       sbp('okTurtles.data/set', 'JOINING_GROUP', true)
       try {
         await sbp('gi.actions/group/joinAndSwitch', {
-          groupId,
-          inviteSecret: secret
+          contractID: groupId,
```

Comment:
Are we going from groupId to contractID?

Reply:
The `groupId` and the `contractID` are the same thing, i.e. a `contractID` can be an ID for any kind of contract, and `groupId` is the `contractID` of a specific Group contract that we're joining